chef/cookbooks/crowbar/templates/default/crowbar-prepare-repositories.sh.erb

```diff
@@ -25,6 +25,8 @@ if [[ -f $UPGRADEDIR/crowbar-prepare-repositories-ok ]] ; then
     exit 0
 fi
 
+log "Disabling RIS services..."
+zypper services --show-enabled-only | egrep ' ris( |$)' | cut -d' ' -f1 | xargs -i zypper modifyservice --disable {}
```

Comment:
I'm still not sure about the grepping. You're probably right. We can be more precise with checking the value of the type column I guess.

(Not supported by zypper, bash magic required)

Reply:
I was surprised that zypper doesn't have `--type` option to apply changes based on service type while it has `--media-type` option which filters by url scheme. I don't have any better idea for filtering out those services. We could try `--xmlout` but without proper xpath utils it would probably be even more hcky than regular table grep.

Reply:
oh, wait, we have `xml_grep`... let me try that.